### ISSUE TITLE
Plattform missing for multiple Assays 

### ISSUE THREAD

**Issue by sneumann** [ISA-tools/mzml2isa#33]
Hi, when running on a set of mzML files, some in pos, some in neg, 
mzml2isa creates two assays. The first is correctly shown:

```
Assay 1
Assay file name: a_MTBLSX_POS_metabolite_profiling_mass_spectrometry.txt
Technology: mass spectrometry
Platform: micrOTOF-Q
```
The second has an empty Plattform:
```
Assay  2
Assay file name: a_MTBLSX_NEG_metabolite_profiling_mass_spectrometry.txt
Technology: mass spectrometry
Platform:
```
I can send some test files in private mail if needed. Yours, Steffen

**Comment by Tomnl** [comment]
Hi Steffen, 

Thanks for the message. That's strange I thought we had fixed this issue. 

Could you send me the files please?

**Comment by Tomnl** [comment]
Also, what version of mzml2isa are you using? 

(The master branch is currently ahead of the releases and conda versions)

**Comment by sneumann** [comment]
Hi, sorry for omitting the version, I have `mzml2isa 0.4.28`
Will try with master later. Yours, Steffen

**Comment by sneumann** [comment]
Indeed, fixed in `mzml2isa 0.5.2` Merci, yours, Steffen
